src/main/webapp/app/exercises/programming/manage/reset/programming-exercise-reset-dialog.component.html

```diff
@@ -0,0 +1,116 @@
+                        </span>
+                    </label>
+                </div>
+            </li>
+            <div class="alert alert-info my-2" *ngIf="this.programmingExerciseResetOptions.deleteBuildPlans && !this.programmingExerciseResetOptions.deleteRepositories">
+                <span jhiTranslate="artemisApp.programmingExercise.reset.deleteBuildPlans.info"> The build plans will be automatically recreated when students submit again. </span>
+            </div>
+            <li>
+                <div class="checkbox">
+                    <label class="control-label">
+                        <input
+                            class="form-check-input"
+                            type="checkbox"
+                            name="deleteRepositories"
+                            [(ngModel)]="this.programmingExerciseResetOptions.deleteRepositories"
+                            (ngModelChange)="this.programmingExerciseResetOptions.deleteBuildPlans = !!$event || this.programmingExerciseResetOptions.deleteBuildPlans"
+                        />
+                        <span jhiTranslate="artemisApp.programmingExercise.reset.deleteRepositories.title" [translateValues]="{ vcPlatform: versionControlName }">
+                            Delete student repositories (on {{ versionControlName }}).
+                        </span>
+                    </label>
+                </div>
+            </li>
+            <li>
+                <div class="checkbox">
+                    <label class="control-label">
+                        <input
+                            class="form-check-input"
+                            type="checkbox"
+                            name="deleteParticipationsSubmissionsAndResults"
+                            [(ngModel)]="this.programmingExerciseResetOptions.deleteParticipationsSubmissionsAndResults"
+                        />
+                        <span jhiTranslate="artemisApp.programmingExercise.reset.deleteParticipationsSubmissionsAndResults">
+                            Delete all student participations, submissions and results (on Artemis).
+                        </span>
+                    </label>
+                </div>
+            </li>
+            <div
+                class="alert alert-warning my-2"
+                *ngIf="!this.programmingExerciseResetOptions.deleteRepositories && this.programmingExerciseResetOptions.deleteParticipationsSubmissionsAndResults"
+            >
+                <span
+                    jhiTranslate="artemisApp.programmingExercise.reset.artefactsWarning"
+                    [translateValues]="{ vcPlatform: versionControlName, ciPlatform: continuousIntegrationName }"
+                >
+                    Selecting Options 3 without all previous options may lead to undeleted artifacts on {{ 'Bitbucket' }} and {{ 'Bamboo' }}. Please ensure this is your intended
+                    action before proceeding.
```

Comment:
I think you forgot to update this text when updating the translation file

Reply:
Yeah also forgot to use the right names for the ci and vc, thanks for noticing!